index.js

```diff
@@ -45,6 +45,30 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
     return arrayOfImportantEvents;
 }
 
+function filterForAuthorAssociation(events = [], association = 'OWNER') {
```

Comment:
I don't think there's a reason that `events` needs a default of `[]`. Events will always be passed in with a value, so I don't think that default would ever get hit and we can take it out.